prow/cmd/deck/static/tide-history/tide-history.ts

```diff
@@ -36,12 +37,15 @@ function optionsForRepoBranch(repo: string, branch: string): Options {
   const hist: {[key: string]: Record[]} = typeof tideHistory !== 'undefined' ? tideHistory.History : {};
   const poolKeys = Object.keys(hist);
   for (const poolKey of poolKeys) {
-    const match = RegExp('(.*?):(.*)').exec(poolKey);
+    // poolKey is in the formatted as `<org>/<repo>:branch`, for example:
+    // - GitHub: `foo/bar:main` # repo: bar
+    // - Gerrit: `https://foo/bar/baz:main` # repo: bar/baz
+    const match = RegExp('(((https|http)://)?.*?):(.*)').exec(poolKey);
```

Comment:
I think the existing regex would get `repo: foo/bar/baz` so either the comment or the regex is wrong?
```suggestion
    // - Gerrit: `https://foo/bar/baz:main` # repo: bar/baz
    const match = RegExp('(((https|http)://[^/]*/)?.*?):(.*)').exec(poolKey);
```

Reply:
done.